### ISSUE TITLE
Different values for metrics in Spring Boot 2.0 compared to 1.5

### ISSUE THREAD

**Issue by robachmann** [micrometer-metrics/micrometer#571]
Hi Jon, 

Maybe this one is related to the issue here where your solution was a ring buffer implementation: https://github.com/micrometer-metrics/micrometer/issues/284 

After updating an application to Spring Boot 2.0.1, I see a previously _lively_ line asymptotically reaching a certain value in Grafana:

Spring Boot 1.5: 
![springboot15](https://user-images.githubusercontent.com/8127370/39124966-ab6b015a-46fd-11e8-8ee3-b7da755784af.png)

Spring Boot 2.0 (see blue and yellow line): 
![springboot20](https://user-images.githubusercontent.com/8127370/39124967-ab847ba8-46fd-11e8-8e3b-d9bfdcfeb081.png)

From my limited knowledge about Micrometer, I would say the ring buffer doesn't overwrite old values and just accumulates them. But maybe it's something completely different :-) 

This happens with the following versions of dependencies:
```
Spring Boot '2.0.1.RELEASE'
compile('org.springframework.boot:spring-boot-starter-web')
compile('org.springframework.boot:spring-boot-starter-actuator')
compile('io.micrometer:micrometer-registry-prometheus:1.0.3')
compile('io.prometheus:simpleclient_pushgateway:0.3.0')
compile('io.prometheus:simpleclient_spring_web:0.3.0')
```

The first image was taken using Spring Boot 1.5.10 together with the Micrometer-Legacy dependency.

I explicitely send only the 0.5 and 0.95 percentile to Prometheus:
```
management:
  metrics.distribution.percentiles:
    all: 0.5, 0.95
```

Could you please have a look at it and state whether I'm doing something wrong? 

Thank you very much and best,
Roman

**Comment by jkschneider** [comment]
I think this is a duplicate of #562. Could you try out 1.1.0-SNAPSHOT and see if it is any better?

**Comment by robachmann** [comment]
Dear Jon, thank you very much for your quick response. I can confirm that 
```
compile('io.micrometer:micrometer-core:1.1.0-SNAPSHOT')
compile('io.micrometer:micrometer-registry-prometheus:1.1.0-SNAPSHOT')
```
remediates the observed behavior. My apologies for the duplicate. 
I'm looking forward to the release of 1.1.0 :-) 

Thank you again and best,
Roman

**Comment by jkschneider** [comment]
@robachmann This will be going into the 1.0.4 release tonight. Just wanted to give you a heads up so you aren't waiting on a minor release that will come much later.

**Comment by robachmann** [comment]
That's very kind of you, thank you for the heads-up.
